infrastructure/lib/infrastructure-stack.ts

```diff
@@ -0,0 +1,54 @@
+import * as ec2 from "@aws-cdk/aws-ec2";
+import * as ecs_patterns from "@aws-cdk/aws-ecs-patterns";
+import * as cdk from "@aws-cdk/core";
+import * as ecs from "@aws-cdk/aws-ecs";
+
+export class InfrastructureStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // TODO we need to figure out best practices for standing up the DB
+    // either in the same CF stack, or in a different one.
+    // what's more, we're going to need to figure out how to create users
+    // in it, and then how to get it initialized to have all of the tables
+    // created for us.
+    // we may be able to use this dynamoDB creation example as a starter: https://github.com/aws-samples/aws-cdk-examples/blob/18f3429414cc80223a04a04d0249b12a7ae13cbb/typescript/api-cors-lambda-crud-dynamodb/index.ts#L10-L21
+    var dsnUser = process.env.SECRET_DSN_USER || "TODO_setDsnUser";
+    var dsnPw = process.env.SECRET_DSN_PASSWORD || "TODO_setDsnPassword";
```

Comment:
All the examples I've seen use AWS secrets manager to handle the password; I wonder if this is just as good?

Reply:
:shrug: I was following the example [here](https://docs.aws.amazon.com/cdk/latest/guide/get_env_var.html).

Reply:
Seems easier than secrets manager so I'm all for it; was just curious whether it mattered